resources/definitions/flashforge_dreamer_nx.def.json

```
@@ -0,0 +1,48 @@
+{
+    "version": 2,
+    "name": "Dreamer NX",
+    "inherits": "flashforge_base",
+    "metadata": {
+        "visible": true,
+        "author": "Egon",
+        "manufacturer": "Flashforge",
+        "file_formats": "text/x-gcode",
+        "platform": "FlashForge_DreamerNX.obj",
+        "platform_offset": [0, 0, 0],
+        "quality_definition": "flashforge_base",
+        "machine_extruder_trains":
+        {
+            "0": "ff_dreamernx_extruder_0"
```

Minor thing, but please rename the extruder to include the full name of the printer
```suggestion
            "0": "flashforge_dreamernx_extruder_0"
```

Also rename the file `resources/extruders/ff_dreamernx_extruder_0.def.json` to `resources/extruders/flashforge_dreamernx_extruder_0.def.json`